src/main/java/homeworks/Infrastructure/wdm/LocalWebDriverFactory.java

```diff
@@ -0,0 +1,10 @@
+package homeworks.Infrastructure.wdm;
+
+public class LocalWebDriverFactory implements WebDriverFactory{
```

Comment:
for each Factory methods(LocalWebDriverFactory/RemoteWebDriverFactory/CloudWebDriverFactory), please add logic same as class "WebDriverManager". Now it is the same for all factories, but will be extended in future

// public String create() {
// String testBrowser = ConfigurationManager.getInstance().getTestBrowser();
// switch (testBrowser) {
// case "chrome":
// return "new Google chrome driver";
// case "firefox":
// return "new Mozilla firefox driver";
// default:
// return "";
// }
// }